presto-main/src/main/java/com/facebook/presto/cost/PlanNodeCostEstimate.java

```diff
@@ -85,13 +138,17 @@ public double getNetworkCost()
      */
     public boolean hasUnknownComponents()
     {
-        return isNaN(cpuCost) || isNaN(memoryCost) || isNaN(networkCost);
+        return isNaN(individualCpuCost) || isNaN(individualMemoryCost) || isNaN(individualNetworkCost) ||
+                isNaN(cpuCost) || isNaN(memoryCost) || isNaN(networkCost);
     }
 
     @Override
     public String toString()
```

Comment:
@rschlussel @Cameronreynoldson Will this change affect the query plan? E.g. is this method used in PlanPrinter?

Reply:
This could probably be changed back to how it was given that the one of the cumulative costs would have to be unknown if any of the individual costs were unknown.